modules/apigee_edge_teams/src/TeamMembershipManager.php

```diff
@@ -90,7 +90,7 @@ public function removeMembers(string $team, array $developers): void {
    */
   public function getTeams(string $developer): array {
     /** @var \Drupal\apigee_edge\Entity\DeveloperInterface $entity */
-    $entity = $this->developerStorage->load($developer);
+    $entity = $this->developerStorage->loadUnchanged($developer);
```

Comment:
We should find a better way to solve this problem, because this solution also eliminates the static cache.